tsl/test/expected/skip_scan.out

```diff
@@ -117,6 +117,11 @@ CREATE INDEX ON :TABLE(time,dev,val);
 :PREFIX SELECT DISTINCT *, 'q1_9' FROM :TABLE ORDER BY dev;
 :PREFIX SELECT DISTINCT dev, time, 'q1_10' FROM :TABLE ORDER BY dev;
 :PREFIX SELECT DISTINCT dev, NULL, 'q1_11' FROM :TABLE ORDER BY dev;
+-- distinct on expressions not supported
+:PREFIX SELECT DISTINCT time_bucket(10,time), 'q1_12' FROM :TABLE;
```

Comment:
Could add a test case where the table has an INDEX on the expression `time_bucket('10,time)` where we can see that SkipScan doesn't get used.

Reply:
there is a test for expression index couple lines above

Reply:
Ah, ok. LGTM